FusionMCPBridge/core/router.py

```
@@ -0,0 +1,419 @@
+            status_code: HTTP status code
+            message: Error message
+            
+        Returns:
+            Error response dictionary
+        """
+        # Check if custom error handler exists
+        if status_code in self.error_handlers:
+            try:
+                return self.error_handlers[status_code](status_code, message)
+            except Exception as e:
+                logger.error(f"Error handler failed: {str(e)}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)